other-examples/collector/hcp-consul/README.md

```diff
@@ -0,0 +1,32 @@
+# HCP Consul OpenTelemetry metrics example setup
+
+This example shows a setup for running a prometheus OpenTelemetry Collector in a docker container to scrape metrics from HCP Consul and post them the New Relic OTLP Collector Endpoint.
+
+**NOTE**: This is *not* for self-managed Consul, and will only work with HCP managed Consul. For self-managed Consul you will have to communicate with the [agent API](https://developer.hashicorp.com/consul/api-docs/agent).
```

Comment:
With the confluent example, we linked out to a page in our docs which explains how to install a dashboard based on these metrics and how to query for these metrics. Is there any equivalent for this?